docs/rules/max-params.md

```diff
@@ -1,4 +1,4 @@
-# Limit Maximum Number of Parameters (max-params)
+# enforce a maximum number of parameters in `function` definitions (max-params)
```

Comment:
Can you add this to the issue with rule descriptions to reconsider in the future?

Because the rule applies to arrow functions too, might need to remove backticks from function to mean the concept instead of the keyword.